pages/index.tsx

```diff
@@ -94,23 +132,19 @@ export default function Home() {
   );
 }
 
-function FcFrame() {
-  return (
-    <>
-      <meta property="fc:frame"
-      content="vNext" /><meta property="fc:frame:image:aspect_ratio"
-      content="1.91:1" /><meta property="fc:frame:image"
-      content="https://unlock-protocol-calendar.vercel.app/api/start-image" /><meta
-      property="og:image"
-      content="https://unlock-protocol-calendar.vercel.app/api/start-image" /><meta
-      property="og:title" content="Unlock Protocol Advent Calendar" /><meta
-      property="fc:frame:post_url"
-      content="https://unlock-protocol-calendar.vercel.app/api?initialPath=%252Fapi&amp;previousButtonValues=%2523A_" /><meta
-      property="fc:frame:button:1" content="Start" /><meta
-      property="fc:frame:button:1:action" content="post" /><meta
-      property="fc:frame:button:1:target"
-      content="https://unlock-protocol-calendar.vercel.app/api/calendar?initialPath=%252Fapi&amp;previousButtonValues=%2523A_" /><meta
-      property="frog:version" content="0.18.2" />
-    </>
-  );
+//Using App directory you don't need to manually create the meta tags
+/*
```

Comment:
Is this comment truly useful here?

Reply:
I had talked with julien about changing the app to use App directory. But eventually didn't because that would mean changing all current components to have `use client` directive. 

In the current system, if for any reason, the frame code is changed, you'll need to change the meta tags manually as well. But if app dir and the `getFrameMetadata` is used, you don't need to change anything in the main app. it just works.